test/integration/__snapshots__/EnvironmentSelect.test.tsx.snap

```diff
@@ -0,0 +1,79 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
to be honest, I think that snaphosts for such large components which include a component from a 3rd party library does not bring much value